standard/stream_links.lua

```diff
@@ -51,4 +55,54 @@ function StreamLinks.readCountdownStreams(args)
 	return stream
 end
 
+--[[
+Resolves the value of a stream given the platform
+]]
+function StreamLinks.resolve(platformName, streamValue)
+	local streamLink = mw.ext.StreamPage.resolve_stream(platformName, streamValue)
+
+	return string.gsub(streamLink, 'Special:Stream/' .. platformName, '')
+end
+
+--[[
+Extracts the streaming platform args from an argument table or a nested stream table inside the arguments table.
+Uses variable fallbacks and resolves stream redirects.
+]]
+function StreamLinks.processStreams(args)
+	local streams = {}
+	if type(args.stream) == 'table' then
+		streams = args.stream
+	end
+
+	for _, platformName in pairs(StreamLinks.countdownPlatformNames) do
+		-- stream has no platform and might be a table or a string
```

Comment:
When can it be a table? I don't recall seeing that being saved

Reply:
see line 73-75
also most /Customs have: `match.stream = match.stream or {}`
which makes it so that `match.stream` might be a table
if that was wrong in the current /CUstom modules we can of course remove the table handling here (plus remove lines 73-75)

Reply:
I could be wrong, but I thought this was about the platform called "stream" (i.e. `		stream = Logic.emptyOr(match.stream.stream, Variables.varDefault('stream')),` in customs), not the match.stream table contain all the platforms?

Reply:
the old code is
```lua
	match.stream = match.stream or {}
 	match.stream = {
 		stream = Logic.emptyOr(match.stream.stream, Variables.varDefault('stream')),
```
which doesn't allow `|stream=link to stream` directly in the input but rather has to have `|stream=table of streams` or `|stream=` unset

i think the most realistic chance of args.stream being already a table is when the match is on a sep. page and the matchlist/bracket gets build from query by id.
i can dm you an example usecase

Reply:
did that now (tested via /dev modules)